user-guide/Advanced_Functionality/DataMiner_Systems/Removing_a_DataMiner_Agent_from_a_DataMiner_System.md

```diff
@@ -19,7 +19,12 @@ Depending on whether the DataMiner Agent is part of a Failover pair or not, a di
 
 1. In the confirmation box, click *Yes*.
 
-1. If you want to keep using the removed DMA as a standalone Agent, restart the DMA.
+1. Stop the Agent you have removed and decommission it.
+
+1. In case your system is cloud connected, go to [Admin.Dataminer.Services](https://admin.dataminer.services/) and navigate further to the DxMs page. Here you see an overview of all the agents with an option to remove a node from the Cloud system. Click the remove button next to the agent you have just removed and confirm.
+
+1. Restart your Cube to have a correct overview on all places.
```

Comment:
What do you mean with "to have a correct overview on all places"?

Reply:
Well, we noticed that for example the BPA UI could still show the old agent until you reconnect with Cube. Other places might have the same, although we haven't found any, I wanted to be sure and just mentioned it as 'all places'.